### ISSUE TITLE
Drop-down for start rule in workbench.

### ISSUE THREAD

**Issue by otac0n** [otac0n/Pegasus#69]
Debugging a specific rule in the workbench is currently achieved via:

```
//debug = #STATE{ /*setup*/ } ruleToDebug
grammar = ...
...
ruleToDebug = ...
```

But a drop-down would be easier.

This should be done as an observable input to the parse result.

**Comment by otac0n** [comment]
Now that `-public` and `-export` are implemented, the work bench needs to be updated to test each of these.

A drop down that allows selection of any of the public of the exported rules, as well as the start rule (if it exists) would be ideal.

**Comment by otac0n** [comment]
Implemented in 873b4537775f74e5d74a35f4c7fea1934ec19b72
